package.json

```diff
@@ -16,13 +16,13 @@
     "node": "18.x"
   },
   "scripts": {
-    "build": "yarn && rm -rf client/dist/* && NODE_ENV=production webpack --mode production --bail --progress",
+    "build": "yarn && yarn lint && rm -rf client/dist/* && NODE_ENV=production webpack --mode production --bail --progress",
     "dev": "NODE_ENV=development webpack --progress",
     "watch": "NODE_ENV=development webpack --watch --progress",
     "lock": "npm-shrinkwrap --dev",
     "test": "NODE_PATH=\"./client/src\" jest",
     "coverage": "NODE_PATH=\"./client/src\" jest --coverage",
-    "lint": "eslint client/src & sass-lint -v"
+    "lint": "eslint client/src"
```

Comment:
Why remove sass-lint? Even if this module currently has no sass, we should leave it in (unless that causes errors) so we don't have to _think to add it_ in the future.

Arguably linting doesn't matter for this module anyway but if we're doing it at all we should do it the standard way.

Reply:
Added back in